.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - 'release-[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetches all tags
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::$(git describe --tags --abbrev=0 HEAD)
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I guess we need to add this secret?

Reply:
Oh, yes you're right. Is that something you can fix?

Reply:
We should already have that token: https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret